src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncReceiptsRequestHandler.scala

```diff
@@ -113,12 +116,14 @@ class FastSyncReceiptsRequestHandler(
   }
 }
 
+// todo move inserting receipts to FastSync
```

Comment:
this todo is no longer relevant because of fast sync refactoring, it will be removed after merge with refactoring branch